mappings/net/minecraft/block/entity/BeeHiveBlockEntity.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/class_4482 net/minecraft/block/entity/BeeHiveBlockEntity
+	CLASS class_4483
+		FIELD field_20425 entityData Lnet/minecraft/class_2487;
+	FIELD field_20423 bees Ljava/util/Map;
+	FIELD field_20424 flowerPos Lnet/minecraft/class_2338;
+	METHOD method_21848 tryEnterHive (Lnet/minecraft/class_1297;Z)V
+	METHOD method_21849 tryEnterHive (Lnet/minecraft/class_1297;ZI)V
+	METHOD method_21856 isFullOfBees ()Z
+	METHOD method_21857 isOriginNotFlower ()Z
```

Comment:
What's the origin?

Reply:
The world origin, (0,0,0). The hive sets its `flowerPos` to that by default. I've changed it to `hasFlowerPos`.